.circleci/config.yml

```diff
@@ -57,21 +57,13 @@ jobs:
           name: reinstall mysql2 temporary
```

Comment:
Remove the coverage steps from Circle-CI to avoid confusing things when it receives multiple files per run. Can remove this completely once merged to master and we can disable circle CI completely.